.env.dist

```diff
@@ -5,6 +5,9 @@ TIDE_API_PROTOCOL=http
 TIDE_API_KEY=<YOUR_API_KEY>
 TIDE_API_SECRET=<YOUR_API_SECRET>
 TIDE_API_VERSION=v1
+# User in Tide API to be credited for project.
+# Omit to user authenticated client user.
+TIDE_API_CREDITED_USER=<LOGIN_NAME>
```

Comment:
So this would be set manually to the `audit_server` user, and if left empty would be what user? I don't see a fallback anymore.

Reply:
It defaults to the authenticated user if empty. 

```
RequestClient string   `json:"request_client,omitempty"`
```